engine/consensus/dkg/messaging_engine_test.go

```diff
@@ -44,8 +48,10 @@ func createTestEngine(t *testing.T) *MessagingEngine {
 // TestForwardOutgoingMessages checks that the engine correctly forwards
 // outgoing messages from the tunnel's Out channel to the network conduit.
 func TestForwardOutgoingMessages(t *testing.T) {
-	// sender engine
 	engine := createTestEngine(t)
+	ctx, cancel := irrecoverable.NewMockSignalerContextWithCancel(t, context.Background())
```

Comment:
I understand that we have two tests only but maybe there will be more in future. I would consider introducing a suite at least to manage context in setup and teardown events. Totally optional though.

Reply:
Done in https://github.com/onflow/flow-go/pull/4219/commits/01f0f83d0f17ca275bc290cafc7fae93832468f0